src/js/components/MyLoopNavBar.jsx

```diff
@@ -52,14 +52,24 @@ class MyLoopNavBar extends React.Component {
 		this.setState({open: !this.state.open, scrolled: scrolled});
 	}
 
+	scrolltoHowitworks () {
+		// Wait a moment for MyPage to get rendered so #howitworks-section exists
+		window.setTimeout(() => {
+			document.getElementById('howitworks-section').scrollIntoView({behavior: 'smooth'});
+		}, 50);
```

Comment:
Good helpful commenting :)
I wonder if that `50` is a race condition? I will test whether `0` or `1` works.

Reply:
Actually, is the setTimeout ever needed? This is a callback to a NavLink click.

Reply:
This is written by @roscoemcinerney, I it should be necessary for implementing the scroll function into the existing codes.